### PR TITLE
Issue #3100072

### PR DESCRIPTION
**Pull request by ethanteague**
Add config for username and password to GovDelivery. 

For https://www.drupal.org/project/govdelivery_bulletins/issues/3100072

### CODE REVIEW

File: src/Form/GovDeliveryBulletinsAdminForm.php [hunk 1]
```diff
@@ -55,6 +54,28 @@ public function buildForm(array $form, FormStateInterface $form_state) {
       '#default_value' => $config->get('api_queue_trigger_enabled'),
     ];
 
+    $form['govdelivery_username'] = [
+      '#type' => 'textfield',
+      '#title' => $this->t('GovDelivery Username'),
+      '#description' => $this->t(
+        'Stores GovDelivery username in database - THIS IS NOT RECOMMENDED.
+        Preferred method is to store in local.settings.php - see README for
```
Comment (ElijahLynn) [inline src/Form/GovDeliveryBulletinsAdminForm.php]:
```suggestion
        Preferred method is to store in settings.local.php - see README for
```

File: src/Form/GovDeliveryBulletinsAdminForm.php [hunk 2]
```diff
@@ -55,6 +54,28 @@ public function buildForm(array $form, FormStateInterface $form_state) {
...
+      '#weight' => '5',
+      '#default_value' => $config->get('govdelivery_username'),
+    ];
+
+    $form['govdelivery_password'] = [
+      '#type' => 'password',
+      '#title' => $this->t('GovDelivery Password'),
+      '#description' => $this->t(
+        'Stores GovDelivery password in database - THIS IS NOT RECOMMENDED.
+        Preferred method is to store in local.settings.php - see README for
```
Comment (ElijahLynn) [inline src/Form/GovDeliveryBulletinsAdminForm.php]:
```suggestion
        Preferred method is to store in settings.local.php - see README for
```

File: README.md [hunk 3]
```diff
@@ -42,3 +42,21 @@ To add a test, it would look like this.
 ```
 
 [GovDelivery Bulletins API](https://developer.govdelivery.com/api/comm_cloud_v1/Default.htm#API/Comm%20Cloud%20V1/API_CommCloudV1_Bulletins_CreateandSendBulletin.htm)
+
+## How to securely store GovDelivery username and password.
+
+Should be stored in local.settings.php, like so:
```
Comment (ElijahLynn) [inline README.md]:
```suggestion
Should be stored in settings.local.php, like so:
```

Comment (swirtSJW) [reply README.md]: I think that works for our implementation... but the more common implementation for others would likely be local.settings as that is usually the one to be used.

**Comment by ElijahLynn** [comment]
https://www.drupal.org/project/govdelivery_bulletins/issues/3100072

File: README.md (hunk 3 above)
Comment (swirtSJW) [reply README.md]: Good catch

**Review by swirtSJW — APPROVED** [review]
Looks good.

**Pull request merged**